deploy/store_all_for_multiple_choice_fields.psql

```diff
@@ -0,0 +1,1069 @@
+-- Deploy afya-tek:store_all_for_multiple_choice_fields to pg
+-- requires: add_normal_views
+-- requires: add_public.combine_field_values_function
+
+BEGIN;
+
+SET search_path TO afyatek_final;
```

Comment:
We should add comments somewhere here for each view indicating how it changes in this migration.

Reply:
> We should add comments somewhere here for each view indicating how it changes in this migration.

This is done.